From e9b96b4b842046c5271efd1db04b1433f06b8e9b Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Tue, 13 Feb 2018 23:06:58 +0530
Subject: [PATCH 0266/1852] v4l: xilinx: csi2rxss: Add rev 3.0 compatible
 string

commit ead7d1664d378ee422f1058b57d38d7c61b86526 from
https://github.com/Xilinx/linux-xlnx.git

Add 3.0 compatible string for new IP version.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-csi2rxss.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/platform/xilinx/xilinx-csi2rxss.c b/drivers/media/platform/xilinx/xilinx-csi2rxss.c
index 708ca7fd7825..7e8f3cf0ce74 100644
--- a/drivers/media/platform/xilinx/xilinx-csi2rxss.c
+++ b/drivers/media/platform/xilinx/xilinx-csi2rxss.c
@@ -1746,6 +1746,7 @@ static SIMPLE_DEV_PM_OPS(xcsi2rxss_pm_ops,
 
 static const struct of_device_id xcsi2rxss_of_id_table[] = {
 	{ .compatible = "xlnx,mipi-csi2-rx-subsystem-2.0" },
+	{ .compatible = "xlnx,mipi-csi2-rx-subsystem-3.0" },
 	{ }
 };
 MODULE_DEVICE_TABLE(of, xcsi2rxss_of_id_table);
-- 
2.31.1

